src/modules/python/pbs/v1/_pmi_types.py

```diff
@@ -120,13 +120,19 @@ def _check_pmi(self):
     def connect(self, endpoint=None, port=None, job=None):
         self._check_pmi()
         if job is None:
-            job = pbs.event().job
+            try:
+                job = pbs.event().job
+            except:
+                pass
```

Comment:
**04/12/2018 14:20:15;0100;pbs_mom;Hook;PBS_power;exechost_startup request rejected by 'PBS_power'**
I found a issue in which mom is reporting a log message at the time of (start|HUP|restart) because in the beginning _connect() is checking for job = pbs.event().job , resulting exechost_startup request rejected by 'PBS_power'. Hence try block is added.